mm2src/mm2_bitcoin/spv_validation/src/storage.rs

```diff
@@ -76,4 +114,6 @@ pub trait BlockHeaderStorageOps: Send + Sync + 'static {
     async fn get_block_height_by_hash(&self, hash: H256) -> Result<Option<i64>, BlockHeaderStorageError>;
 
     async fn remove_headers_up_to_height(&self, to_height: u64) -> Result<(), BlockHeaderStorageError>;
+
+    async fn is_table_empty(&self) -> Result<(), BlockHeaderStorageError>;
```

Comment:
Isn't `BlockHeaderStorageOps::is_table_empty` used in tests only? How about adding the `cfg(not(test))`?

Reply:
yeah it doesn't work as `is_table_empty` is not seen as a member of trait `BlockHeaderStorageOps` when implementing for IDB and SQL storage. even tho they're marked with the `cfg(test)`